src/function.js

```diff
@@ -34,7 +34,7 @@ Sk.builtin.func = Sk.abstr.buildNativeClass("function", {
 
         this.$name = (code.co_name && code.co_name.v) || code.name || "<native JS>";
         this.$d = Sk.builtin.dict ? new Sk.builtin.dict() : undefined;
-        this.$doc = code.$doc;
+        this.$doc = code.$doc || Sk.builtin.none.none$;
```

Comment:
I don't mind this - I actually prefer what had before - e.g. `co_docstring` just because it feels more inkeeping with other `co_*` objects.

so
```javascript
this.$doc = code.co_docstring || Sk.builtin.none.none$;
```

context: I think this was a place holder and was similar to the `sk_method` api. But the apis are separate so they can deviate here.

Reply:
Happy for it to be `co_docstring`.  I've updated the PR and force-pushed.